### PR TITLE
Show username of online users.

### PR DESCRIPTION
**Pull request by rs4231199**
Screenshot from 2019-07-22 15-05-24

### CODE REVIEW

**Comment by abhigyank** [comment]
A lot of changes in `client.js` is unnecessary, it seems to be some shift in alignment. Can you undo those? @rs4231199

File: public/client.js [hunk 1]
```diff
@@ -59,18 +67,20 @@ socket.on('user set', function (data) {
     $("body").css("background-color","#f8f8f8");
     $(".wrapper").fadeIn();
     $(".chat[data-chat='person1']").append("<div class='conversation-start'>\
-                                                <span>" + date.getHours() + ':' + date.getMinutes() + "</span>\
-                                            </div>");
+    					        <span>" + date.getHours() + ':' + date.getMinutes() + "</span>\
```
Comment (abhigyank) [inline public/client.js]: I don;t understand why all these are coming as change.
Have you changed tabs to spaces or spaces to tabs?

**Comment by abhigyank** [comment]
The logic of the implementation looks correct to me. I'll do a thorough review soon.

File: public/client.js (hunk 1 above)
Comment (rs4231199) [reply public/client.js]: Yes, I changed that in sublime text, but don't know how to fix. A help would be appreciated.

**Comment by rs4231199** [comment]
@abhigyank I have one more branch which doesn't have any such indentation problem, if you want I can create another pull request or hard-reset master branch to that commit.

**Pull request closed without merging**